### PR TITLE
Update group and artifact IDs, update parent

### PR DESCRIPTION
**Pull request by pavelhoral**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
```
Comment (kovaceviccz) [inline .github/workflows/build.yml:12]: This could be used as an opportunity to add support for JDK 17.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,23 @@
+# 3.0.0
+
+### Major
+
+* Switched to Wren group and artifact ids
+
+
 # 2.0.0
 
 ## Changes
```
Comment (kovaceviccz) [inline CHANGELOG.md:10]: Should be used either nowhere or everywhere.

File: pom.xml [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
-information: "Portions Copyrighted [year] [name of copyright owner]".
+    When distributing Covered Software, include this CDDL Header Notice in each file and include
+    the License file at legal/CDDLv1.0.txt. If applicable, add the following below the CDDL
+    Header, with the fields enclosed by brackets [] replaced by your own identifying
+    information: "Portions Copyrighted [year] [name of copyright owner]".
 
-Copyright 2014-2015 ForgeRock AS.
-Portions Copyright 2017 Wren Security.
+    Copyright 2014-2015 ForgeRock AS.
+    Portions Copyright 2017-2021 Wren Security.
```
Comment (kovaceviccz) [inline pom.xml:16]: Is outdated.

File: wrensec-guice-core/src/test/java/org/forgerock/guice/core/test/TestModule1.java [hunk 4]
```diff
@@ -22,7 +22,6 @@
 public class TestModule1 implements Module {
 
     private TestModule1() {
-
```
Comment (kovaceviccz) [inline wrensec-guice-core/src/test/java/org/forgerock/guice/core/test/TestModule1.java:25]: This seems unnecessary. Why hasn't a change of this kind been made in other modules too?

File: CHANGELOG.md (hunk 2 above)
Comment (pavelhoral) [reply CHANGELOG.md:10]: I would rather have Java 17 support as a separate PR as this one is big as it is already.

File: pom.xml (hunk 3 above)
Comment (pavelhoral) [reply pom.xml:16]: This is correct as the PR is from the last year.

File: .github/workflows/build.yml (hunk 1 above)
Comment (pavelhoral) [reply .github/workflows/build.yml:12]: I would rather have Java 17 support as a separate PR as this one is big as it is already.

File: CHANGELOG.md (hunk 2 above)
Comment (pavelhoral) [reply CHANGELOG.md:10]: We need to decide what to do with this file after merging the project in wrensec-commons.

File: wrensec-guice-core/src/test/java/org/forgerock/guice/core/test/TestModule1.java (hunk 4 above)
Comment (pavelhoral) [reply wrensec-guice-core/src/test/java/org/forgerock/guice/core/test/TestModule1.java:25]: I understand that the change is a bit random, but I am all for fixing incorrect code formatting when possible.

**Pull request merged**